src/network/core_utils.rs

```diff
@@ -911,7 +922,16 @@ impl CoreUtils {
                 ))
             }
         }
-
+        let k = format!("/proc/sys/net/ipv6/conf/{}/enhanced_dad", &host_veth);
```

Comment:
This is the wrong sysctl, you need to set `accept_dad` not `enhanced_dad`

Reply:
Also you have set both sysctls for the bridge interface as well not only the veth